src/GiftedChat.js

```diff
@@ -573,6 +586,7 @@ GiftedChat.propTypes = {
   loadEarlier: PropTypes.bool,
   onLoadEarlier: PropTypes.func,
   isLoadingEarlier: PropTypes.bool,
+  renderKeyboardSpacer: PropTypes.func,
```

Comment:
Why this props if you don't use it?

Reply:
I use it in the _render()_ function of GiftedChat.js

```javascript
render() {
  if (this.state.isInitialized === true) {
      return (
        <View style={styles.container}>
          <ActionSheet ref={(component) => (this._actionSheetRef = component)}>
            <View style={styles.container} onLayout={this.onMainViewLayout}>
              {this.renderMessages()}
              {this.renderInputToolbar()}
            </View>
          </ActionSheet>
         {this.renderKeyboardSpacer()}
        </View>
      );
    }
```